<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindInspector</class>
 <widget class="QWidget" name="BindInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="cbid_label">
     <property name="text">
      <string>CB ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="params_label">
     <property name="text">
      <string>parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="remove_button">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
